hooks/shared/useGetRequest/useGetRequest.tsx

```diff
@@ -84,6 +84,8 @@ function useGetRequest<T = unknown>(uri?: string, options?: Options): State<T> {
         return;
       }
 
+      cache.current[url!] = json;
```

Comment:
Oh, thanks for fixing this Alex I just realized our cache won't work for requests nor requiring an access token

Reply:
not*

Reply:
Now the cache should work when access token isn't required, Rashad.